docs/contributors/code/react-native/osx-setup-guide.md

```diff
@@ -105,37 +92,46 @@ In another terminal type:
 npm run native ios
 ```
 
-After waiting for everything to build we should see:
+After waiting for everything to build, the demo app should be running from the iOS simulator:
```

Comment:
Nice one, it's better to be more explicit on what the image below is supposed to show.